src/net/sf/memoranda/ui/AppFrame.java

```diff
@@ -944,13 +951,67 @@ public void doImportCal() {
         if (chooser.showOpenDialog(this) != JFileChooser.APPROVE_OPTION){
             return;
         }
-        Context.put("LAST_SELECTED_PACK_FILE", chooser.getSelectedFile());        
-        java.io.File f = chooser.getSelectedFile();
+        Context.put("LAST_SELECTED_ICS_FILE", chooser.getSelectedFile());        
+        final java.io.File f = chooser.getSelectedFile();
         CalendarICS.importCalendar(f);
         projectsPanel.prjTablePanel.updateUI();
 		
 	}
     
+    /**
+     * Assembles browser for exporting project as ics file
+     */
+    public void doExportCal(){
+    	UIManager.put("FileChooser.saveInLabelText", Local.getString("Save in:"));
+        UIManager.put("FileChooser.upFolderToolTipText", Local.getString(
+                "Up One Level"));
+        UIManager.put("FileChooser.newFolderToolTipText", Local.getString(
+                "Create New Folder"));
+        UIManager.put("FileChooser.listViewButtonToolTipText", Local.getString("List"));
+        UIManager.put("FileChooser.detailsViewButtonToolTipText", Local.getString("Details"));
+        UIManager.put("FileChooser.fileNameLabelText", Local.getString(
+                "File Name:"));
+        UIManager.put("FileChooser.filesOfTypeLabelText", Local.getString(
+                "Files of Type:"));
+        UIManager.put("FileChooser.saveButtonText", Local.getString("Save"));
+        UIManager.put("FileChooser.saveButtonToolTipText", Local.getString(
+                "Save selected file"));
+        UIManager.put("FileChooser.cancelButtonText", Local.getString("Cancel"));
```

Comment:
There are several instances where you call Local.getString(*string literal*) and use a literal instead of a constant. I'm not sure if that is something that's worth changing, since those are pretty conventional button names. However, I also know that if for ever some weird reason somebody wanted to change the text of those buttons, that would take a little searching. I just don't know if it's really worth it here, though.